application/src/cdnn/src/cdnn_main.c

```diff
@@ -96,6 +96,7 @@ static int R_FC_SyncStartWrapperAI();
 static int R_FC_SyncEndWrapperAI();
 static void softmax(float *x, int n);
 char * gp_ai_final_buffer;
+void Conv_YUYV2RGB(unsigned char * bgr, unsigned char * yuyv, int width, int height);
```

Comment:
Are we supposed to have this function as `Conv_YUYV2RGB()` so that we can distinguish from `Conv_YUYV2RGB()`?

Reply:
Should be reverted. format should be func_name(src, dest, width, height)